CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing to Outstatic
+
+- Before jumping into a PR be sure to search [existing PRs](https://github.com/avitorio/outstatic/pulls) or [issues](https://github.com/avitorio/outstatic/issues) for an open or closed item that relates to your submission.
+- If you can't get something to work the way you expect or found a bug, [open a question](https://github.com/avitorio/outstatic/discussions/new?category=q-a) in our discussion forums.
+- If you are interested in adding a feature or improving the code, please [start a discussion](https://github.com/avitorio/outstatic/discussions/new?category=ideas) about it first.
+
```

Comment:
Suggestion: Explain the package structure (what's in /apps, vs /package vs /example)

Reply:
Done